src/components/Admin/StoriesForApproval.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import { Spinner, Card, Media, Badge, Row } from 'reactstrap';
+import _ from 'lodash';
+import '../../styles/TrendingStories.css';
+import axios from 'axios/index';
+import { TRUTHTREE_URI } from '../../constants';
+import { connect } from 'react-redux';
+import ReactHtmlParser from 'react-html-parser';
+
+class StoriesForApproval extends Component {
+  constructor(props) {
+    super(props);
+    this.getStoryDetails = this.getStoryDetails.bind(this);
+    this.selectStory = this.selectStory.bind(this);
+    this.state = {
+      InitialData: [],
+      bgColor: []
+    };
+  }
+
+  contentHtml(data) {
+    return ReactHtmlParser(data);
+  }
+
+  componentDidMount() {
+    //List of stories to be approved if admin
+    axios
+      .get(`${TRUTHTREE_URI}/api/stories/pending`)
+      //Change the api call to unapproved stories
+      .then(response => {
+        let color = [];
+        for (var i = 0; i < response.data.length; i++) {
+          color.push('white');
+        }
+        this.props.dispatch({
+          type: 'STORIES_LIST',
+          adminStories: response.data,
+          adminStoriesLength: response.data.length,
```

Comment:
Why does the length need to be in a separate property? Can we make use of `adminStories.length`?

Reply:
I am using it in many places, hence storing it as such